master/pyext/src/validation/__init__.py

```diff
@@ -16,6 +16,42 @@
 from collections import defaultdict
 from validation import utility
 
+import logging
+from typing import Final
```

Comment:
typing.Final needs Python 3.8 or later. That's probably OK in this case since most usage will be in a venv or container, but I would be careful about introducing any unnecessary dependencies.

Reply:
Got it. As a side note - Python 3.8 is already mentioned in the README:

>Create and activate a Python3.8 virtual environment.

I think it wouldn't hurt if we explicitly define and freeze the Python version (especially while we are in the beta stage). Judging from https://endoflife.date/python 3.8 should have enough support time until PDB-Dev is merged into upstream.